metadb/component.go

```diff
@@ -0,0 +1,159 @@
+package metadb
```

Comment:
Seems like we won't have other *.go files other than the component.go for metadb

so how about component/metadb.go?  later we may have component/smtp.go and etc

Reply:
SG. It is extracted to become a separate package because the circular dependency problem will be encountered after the Profile is moved to the server package.